Shared/Samples/Generate offline map with custom parameters/GenerateOfflineMapWithCustomParametersView.CustomParameters.swift

```diff
@@ -0,0 +1,156 @@
+        /// the download.
+        @State private var includeSystemValves = true
+        
+        /// A Boolean value indicating if the service connections layer should be included
+        /// in the download.
+        @State private var includeServiceConnections = true
+        
+        /// The minimum flow rate by which to filter features in the Hydrants layer,
+        /// in gallons per minute.
+        @State private var minHydrantFlowRate = 0.0
+        
+        /// The hydrant flow rate range.
+        private let hydrantFlowRateRange = 0.0...1500.0
+        
+        /// A Boolean value indicating if the pipe layers should be restricted to
+        /// the extent frame.
+        @State private var shouldCropWaterPipesToExtent = true
+        
+        var body: some View {
+            NavigationStack {
+                Form {
+                    Section(header: Text("Adjust Basemap")) {
+                        HStack {
+                            Text("Min Scale Level")
+                            Spacer()
+                            Text(minScaleLevel, format: .number.precision(.fractionLength(0)))
+                        }
+                        Slider(value: $minScaleLevel, in: scaleLevelRange, step: 1.0)
+                        
+                        HStack {
+                            Text("Max Scale Level")
+                            Spacer()
+                            Text(maxScaleLevel, format: .number.precision(.fractionLength(0)))
+                        }
+                        Slider(value: $maxScaleLevel, in: scaleLevelRange, step: 1.0)
+                        
+                        HStack {
+                            Text("Extent Buffer Distance")
+                            Spacer()
+                            Text(basemapExtentBufferDistance, format: .number.precision(.fractionLength(0)))
+                        }
+                        Slider(value: $basemapExtentBufferDistance, in: basemapExtentBufferRange)
+                    }
+                    
+                    Section(header: Text("Include Layers")) {
+                        Toggle("System Valves", isOn: $includeSystemValves)
+                        Toggle("Service Connections", isOn: $includeServiceConnections)
+                    }
+                    
+                    Section(header: Text("Filter Feature Layer")) {
+                        HStack {
+                            Text("Min Hydrant Flow Rate")
+                            Spacer()
+                            Text(minHydrantFlowRate, format: .number.precision(.fractionLength(0)))
+                        }
+                        Slider(value: $minHydrantFlowRate, in: hydrantFlowRateRange, step: 1.0)
+                    }
+                    
+                    Section(header: Text("Crop Layer to Extent")) {
+                        Toggle("Water Pipes", isOn: $shouldCropWaterPipesToExtent)
+                    }
+                }
```

Comment:
```suggestion
                }
                .navigationTitle("")
                .navigationBarTitleDisplayMode(.inline)
```

We generally add a title for sheets in the samples. Something like "Generate Offline Map" could work. That way, it kind of explains what the "Start" button does too.